canonical-web-and-design/cn.ubuntu.com
# Set width and height to thumbnails

## Done

Fix blog article thumbnails.

## QA

https://cn.ubuntu.com/blog
Check the thumbnails, you will notice they do not have the same size.
https://cn-ubuntu-com-436.demos.haus/blog
Check the thumbnails, you will notice they all have the same size(354x199) and cover the whole width of the block.

## Issue

Part of: https://github.com/canonical-web-and-design/canonicalwebteam.blog/pull/153

## Screenshots

Before:
![image](https://user-images.githubusercontent.com/6387619/94825207-cfa15c80-03fd-11eb-828d-5bc054417550.png)

After:
![image](https://user-images.githubusercontent.com/6387619/94825308-e9db3a80-03fd-11eb-8517-f7f92bf582cc.png)
